docs/docs/infrahubctl/infrahubctl-repository.mdx

```diff
@@ -47,6 +48,26 @@ $ infrahubctl repository add [OPTIONS] NAME LOCATION
 * `--config-file TEXT`: [env var: INFRAHUBCTL_CONFIG; default: infrahubctl.toml]
 * `--help`: Show this message and exit.
 
+## `infrahubctl repository init`
+
+Initialize a new Infrahub repository.
+
+**Usage**:
+
+```console
+$ infrahubctl repository init [OPTIONS] DST
+```
+
+**Arguments**:
+
+* `DST`: [required]
```

Comment:
It's not clear what DST is, or what it is that we are supposed to provide.

Reply:
Should DST be optional? If not provided, we could make a directory from the project name that we prompt for.

Reply:
Unfortunately from what I read copier requires the destination to be known upfront before the prompt appears using the `run_copy` function. By default copier will use `cwd` if nothing is passed in. which would require this workflow. We are using package name more for the `pyproject.toml` package name. 

```
mkdir infrahub-repo`
cd infrahub-repo
infrahubctl repository init
```

Rather than just `infrahubctl repository init infrahub-repo`

Reply:
DST now also has help to explain what it is.